### DISCUSSION TITLE
Adding Subdomain

### DISCUSSION THREAD

**Discussion by 0xmaximize** [vercel/community#6627] [Help] [answered]
### Summary

Hello i want to add 2 subdomain on vercel, is it possible?

### Steps to Reproduce

Hello i want to add 2 subdomain on vercel, is it possible?

**Comment by amyegan** [comment, marked as answer]
Hey @0xmaximize! Yes, you can add subdomains. You'll essentially follow the same steps as you would for adding a custom domain

https://vercel.com/docs/projects/domains/add-a-domain#
